ui/components/multichain/app-header/index.scss

```diff
@@ -73,8 +73,8 @@
   }
 }
 
-.multichain-app-header-shadow {
-  box-shadow: var(--shadow-size-md) var(--color-shadow-default);
+.multichain-app-header-border {
+  border-bottom: 1px solid var(--color-border-muted);
```

Comment:
I'm really confused by this change, we use box-shadow for other headers too, e.g. the Snaps confirmation headers. If anything this will make that feel more disconnected. I thought we were trying to use the box-shadow look everywhere?

Perhaps @eriknson can chime in

Reply:
Here is the issue for reference, I'm open to adding this elsewhere if we need to: https://github.com/MetaMask/metamask-extension/issues/27313